context-free.md

```diff
@@ -1,22 +1,27 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+_Anyone who wants to create online generated art._
 
 ## What is easy to do in this language? Why is it easy?
-
+_It is easy to create shapes because you can just type in what you want and it will appear. 
+Also, it is easy to generate variations of the same shape with different parameters._
 
 ## What is hard to do in this language? Why is it hard?
-
+_I found it hard to put the shapes I created exaclty where I wanted. Having to toy with the 
+x and y coordinates was a lot more difficult than just dragging and dropping the shapes where I wanted 
+them to go._
 
 ## How did you learn how to program in this language?
-
+_I looked at examples from my class mates and art that I found online. I also just played with it for
+a while._
 
 ## What is the underlying _computational model_ for this programming language? 
-_We don't yet have a great definition of the term "computational model". 
-For now, try to come up with the clearest, most concise explanation of what 
-happens when a ContextFree program runs._
+_It displays shapes in a given space taking size position and color into account._
 
 
 ## What do you think is interesting about the ContextFree program you wrote?
+_I thought the colors I used were interesting, it took me a while to figure out how to achive the
```

Comment:
I'm curious how you ended up getting that blue. Did you look up  an HSL color picker or just mess with the values?